sections/proof-of-yield/DayBasicData.tsx

```diff
@@ -18,40 +17,33 @@ import { smSize, lgSize, xlSize } from "../../constants";
 import { shortenAddress } from "../../utils";
 import { useViewWidth } from "../../hooks";
 import { twMerge } from "tailwind-merge";
-const { commify } = utils;
-
-const e = {
-  block: 16188461,
-  date: 1675886432000,
-  action: "Rebase",
-  yieldDistributed: 1873.92,
-  fees: 208.21,
-  transactionHash:
-    "0xd9973207c58e3a041786faba972417f853776b88e0453ea113eaa79e44ef4b07",
-};
-
-const mockData = [];
-for (let i = 0; i < 5; i++) {
-  mockData.push(
-    Object.assign(
-      { ...e },
-      { block: e.block + i * 100, date: e.date + i * 1500 }
-    )
-  );
-}
+import { DailyStat, YieldBoostMultiplierProps } from "../../types";
 
 const eventChartColumnCssRight = "pr-6 xl:pr-8";
 const eventChartColumnCssLeft = "pl-6 xl:pr-8";
 
 interface DayBasicDataProps {
   timestamp: number;
+  dailyStat: DailyStat;
   sectionOverrideCss?: string;
 }
 
-const DayBasicData = ({ timestamp, sectionOverrideCss }: DayBasicDataProps) => {
+const DayBasicData = ({
+  timestamp,
+  dailyStat,
+  sectionOverrideCss,
+}: DayBasicDataProps) => {
   const router = useRouter();
   const width = useViewWidth();
 
+  const yieldBonusProps: YieldBoostMultiplierProps = {
+    rawApy: parseFloat(dailyStat.raw_apy),
```

Comment:
Seems like `dailyStat` could be undefined without timestamp parsing. Either create a guard to not render this in the parent usage or to be safe just add in `?` w/ a default value, perhaps `0`